README.md

```diff
@@ -10,10 +10,438 @@ gem 'registers-ruby-client', git: 'https://github.com/openregister/registers-rub
+#### `item_hash`
+
+Gets the SHA-256 hash of the item which the entry points to.
+
+<details>
+<summary>
+Example usage (click here to expand):
+</summary>
+
+```
+
+register_data = registers_client.get_register 'country', 'beta'
+
+entry = register_data.get_entries.select {|entry| entry.key == 'CZ'}.first
+entry.item_hash
+
+```
+</details>
+<details>
+<summary>
+Expected output (click here to expand):
+ </summary>
+
+```
+sha-256:c45bd0b4785680534e07c627a5eea0d2f065f0a4184a02ba2c1e643672c3f2ed
+```
+</details>
+
+#### `value`
+
+Returns the entry as a hash.
+
+<details>
+<summary>
+Example usage (click here to expand):
+</summary>
+
+```
+
+register_data = registers_client.get_register 'country', 'beta'
+
+entry = register_data.get_entries.select {|entry| entry.key == 'CZ'}.first
+entry.value.to_json
+
+```
+</details>
+<details>
+<summary>
+Expected output (click here to expand):
+ </summary>
+
+```
+"{"key":"CZ","timestamp":"2016-04-05T13:23:05Z","item_hash":"sha-256:c45bd0b4785680534e07c627a5eea0d2f065f0a4184a02ba2c1e643672c3f2ed"}"
+```
+</details>
+
+### Record
+
+#### `entry`
+
+Gets the `Entry` object associated with the record.
+
+<details>
+<summary>
+Example usage (click here to expand):
+</summary>
+
+```
+
+register_data = registers_client.get_register 'country', 'beta'
+
+record = register_data.get_records.select {|record| record.entry.key == 'CZ'}.first
+record.entry.to_json
+
+```
+</details>
+<details>
+<summary>
+Expected output (click here to expand):
+ </summary>
+
+```
+"{"entry_number":205,"parsed_entry":{"key":"CZ","timestamp":"2016-11-11T16:25:07Z","item_hash":"sha-256:c69c04fff98c59aabd739d43018e87a25fd51a00c37d100721cc68fa9003a720"}}"
+```
+</details>
+
+#### `item`
+
+Gets the `Item` object associated with the record.
+
+<details>
+<summary>
+Example usage (click here to expand):
+</summary>
+
+```
+
+register_data = registers_client.get_register 'country', 'beta'
+
+record = register_data.get_records.select {|record| record.entry.key == 'CZ'}.first
+record.item.to_json
+
+```
+</details>
+<details>
+<summary>
+Expected output (click here to expand):
+ </summary>
+
+```
+"{"item_json":"{\"citizen-names\":\"Czech\",\"country\":\"CZ\",\"name\":\"Czechia\",\"official-name\":\"The Czech Republic\",\"start-date\":\"1993-01-01\"}","item_hash":"sha-256:c69c04fff98c59aabd739d43018e87a25fd51a00c37d100721cc68fa9003a720","parsed_item":null}"
+```
+</details>
+
+## Collections
+
+The majority of the methods available in the `RegisterClient` (described in more detail in the next section) return one of three types of collection object. These collections all include `Enumerable` and implement the `each` method.
+
+### EntryCollection
+
+A collection of `Entry` objects.
+
+#### `each`
+
+Yields each `Entry` object in the collection.
+
+#### `page(int page=1)`
+
+Returns all `Entry` objects in the collection, according to the specified `page` number (defaults to `1`) and `page_size` (defined when creating the `RegisterClientManager`).
```

Comment:
Included this for completeness, as this is what it's doing, but happy to remove if you think that this will be obvious from the description of the `RegisterClientManager`.

Reply:
I'm happy to remove this.